Feat/modem db 2.0.1 (#435)
NordicSemiconductor/pc-nrfconnect-cellularmonitor: Changelog.md

> ## 2.4.1 - 2024-07-09

Reviewer: unreleased I assume
```suggestion
## 2.4.3 - unreleased
```

## 2.4.3 - unreleased

### Added

> -   Pre-bundled Modem Trace Databases v1.3.6, v1.3.6-FOTA, v1.3.7, v1.3.7-FOTA,
>     v2.0.1, v2.0.1-FOTA, v2.0.2 and v2.0.2-FOTA.

Reviewer 2: ```suggestion
-   Pre-bundled Modem Trace Databases: v1.3.6, v1.3.6-FOTA, v1.3.7, v1.3.7-FOTA,
    v2.0.1, v2.0.1-FOTA, v2.0.2, and v2.0.2-FOTA.
```

### Fixed

> -   Modem Trace Database dropdown filters out unsupported DB for nRF9151.

Reviewer 2: ```suggestion
-   Issue where the Modem Trace Database's drop-down would filter out an unsupported database for nRF9151.
```
How is filtering out an _unsupported_ db an issue? Sounds logical to me to filter out something that is not supported.
